app/Email.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace emailApp;
+
+class Email {
+	function validateEmail($email) {
```

Comment:
function name should be plain verb. You should use verb+noun if you are performing activity on something other than its class.  Repeating class name in function name is not good. You can simply name function `validate`